gprofiler/profilers/perf.py

```diff
@@ -243,6 +244,16 @@ def _get_metadata(self, pid: int) -> Optional[AppMetadata]:
             pass
         return None
 
+    def _get_appid(self, pid: int) -> Optional[str]:
+        if pid in (0, -1):
```

Comment:
Don't need this check here, it's to remove "sentinel" values returned by `perf`, but `_get_metadata` already removed them, so you don't need to check again.

Reply:
(this function won't be called if `pid in (0, -1)`)